static/app/stores/organizationEnvironmentsStore.tsx

```diff
@@ -20,7 +20,7 @@ type State = {
 };
 
 type OrganizationEnvironmentsStoreInterface = {
```

Comment:
Could ou also update this to use the `CommonStoreInterface` https://github.com/getsentry/sentry/blob/72207cffd8f71408e78039fe58d99838d7d03e48/static/app/stores/alertStore.tsx#L8-L23

Reply:
Then the `getState` here goes away in favor of

```tsx
type OrganizationEnvironmentsStoreInterface = CommonStoreInterface<State> &{
  ...
}
```

Reply:
Sure. I find it ironic that we have `type OrganizationEnvironmentsStoreInterface` :D clearly, this should have been an interface lol